src/components/Disclaimer/utils.tsx

```diff
@@ -42,7 +42,7 @@ const metricToSourceMap: RegionSourceMap = {
   },
   [Metric.CASE_GROWTH_RATE]: {
     state: {
-      sourceName: 'The New York Times',
+      sourceName: 'The New York Times!',
```

Comment:
❗ ❓ 

(guessing this was for testing 😁 )

Reply:
whoops hahahaha yes